lib/app/data/providers/secure_storage_provider.dart

```diff
@@ -122,6 +122,22 @@ class SecureStorageProvider {
     );
   }
 
+  // write and read last choosen language
+  Future<String> readCurrentLanguage({required String key}) async {
```

Comment:
Aren't we already storing the locale using get_storage_provider? Best to continue using it for all non-secret information.

Reply:
Understood what's you trying to say, I will remove these data from secure_storage and store into get_store_provider.